content/docs/components-and-props.md

```diff
@@ -241,7 +241,7 @@ Bileşenlerin çıkarılması en başta angarya bir işlem gibi görünebilir. F
 
 Bu bileşen, içerisinden bileşenler çıkarmak için iyi bir adaydır diyebiliriz.
 
-## Prop'lar ve Salt Okunurlar {#props-are-read-only}
+## Prop'lar Salt Okunurlar {#props-are-read-only}
```

Comment:
Since the original title was `props are read-only`, this translation is wrong. But the existing one is also misleading. we can change it to something like:

> Prop'lar Salt Okunurdur

Reply:
You're right. Your Advice seems better than mine. Thank you for feedback.